src/shared/Button.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react'
+import styled from 'styled-components/native'
+import { globalVariables, ButtonTitle } from '../modules/style'
+
+interface TouchableType {
+  onPress: () => void
+  outline?: boolean
+  active?: boolean
+  disabled?: boolean
+  width?: string | number
+  height?: string | number
+  background?: boolean
+  onBlur?: () => void
+}
+
+interface ButtonType extends TouchableType {
+  title: string
+  fontSize?: number
+  warn?: boolean
+}
+
+const Button = ({
+  title,
+  onPress,
+  outline,
+  active,
+  fontSize,
+  disabled,
+  width,
+  height,
+  warn,
+  background = true,
+  onBlur = () => {},
+}: ButtonType) => {
+  if (!title) {
+    console.error('title is required!')
```

Comment:
Do we need a console error here? 🤔

Reply:
oh right haha since we're using TypeScript I'll anyway show error when required prop is missing!